muspinsim_combine/muspinsim_combine.xml

```diff
@@ -4,26 +4,15 @@
         <!-- Don't have MuSpinSim as dependency, but use that to version as need
              to maintain compatability with its output files -->
         <!-- version of underlying tool (PEP 440) -->
-        <token name="@TOOL_VERSION@">2.0.2</token>
+        <token name="@TOOL_VERSION@">2.2.1</token>
         <!-- version of this tool wrapper (integer) -->
         <token name="@WRAPPER_VERSION@">0</token>
         <!-- citation should be updated with every underlying tool version -->
         <!-- typical fields to update are version, month, year, and doi -->
-        <token name="@TOOL_CITATION@">
-            @software{muspinsim,
-                author = {Sturniolo, Simone and Liborio, Leandro and Owen, Josh and Mudaraddi, Anish and Davies, Joel and Wilkinson, John and {Muon Spectroscopy Computational Project}},
-                license = {MIT},
-                title = {{muspinsim}},
-                url = {https://github.com/muon-spectroscopy-computational-project/muspinsim},
-                version = {v2.0.2},
-                month = {1},
-                year = {2023}
-                doi = {10.5281/zenodo.7568830}
-            }
-        </token>
+        <token name="@TOOL_CITATION@">10.5281/zenodo.7733979</token>
```

Comment:
should this token not be shared with all tools instead of redefining it in all tools

Reply:
Thanks for the recommendation, I've refactored this into the shared macros file